chia/cmds/coin_funcs.py

```diff
@@ -217,3 +220,9 @@ async def async_split(args: Dict[str, Any], wallet_client: WalletRpcClient, fing
     tx_id = transaction.name.hex()
     print(f"Transaction sent: {tx_id}")
     print(f"To get status, use command: chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}")
+    if final_amount_per_coin < dust_threshold and wallet_type == WalletType.STANDARD_WALLET:
+        print(
+            f"WARNING: The amount per coin: {amount_per_coin} is less than the dust threshold: {dust_threshold / mojo_per_unit}."
+            f"Some or all of the Coins {'will' if number_of_coins > spam_filter_after_n_txs else 'may'} not show up in your wallet unless you "
+            f"decrease the dust limit to below {final_amount_per_coin} mojos or disable it by setting it to 0."
```

Comment:
It would probably be more helpful to actually name the configuration option here. `xch_spam_amount`